src/Constants/ShippingCountry.php

```diff
@@ -33,4 +33,19 @@ class ShippingCountry
     const POLAND = 23;
     const SWEDEN = 24;
     const SINGAPORE = 25;
+
+    const ADDRESS_FORMAT_DE = 'DE';
+    const ADDRESS_FORMAT_EN = 'EN';
+
+    public static function getAddressFormat($countryId)
+    {
+        switch($countryId)
+        {
+            case self::UNITED_KINGDOM:
+            case self::IRELAND:
```

Comment:
@felixdausch Ich habe das grade einmal geprüft, bei Irland wird in der UI ebenfalls das deutsche Layout angezeigt. Das EN wird nur bei UK angewendet.